static/title-bar.less

```diff
@@ -1,15 +1,17 @@
 @import "ui-variables";
 
 @title-bar-text-size: 13px;
+@title-bar-height: 23px;
 @title-bar-background-color: @base-background-color;
 @title-bar-border-color: @base-border-color;
 
 body.fullscreen .title-bar {
-  display: none;
+  margin-top: -@title-bar-height;
 }
 
 .title-bar {
-  height: 23px;
+  height: @title-bar-height;
+  transition: margin-top 200ms ease-out 700ms;
```

Comment:
Ya, too bad the `.fullscreen` styles stay until after the "fullscreen animation" finishes.

How do you like just

```css
transition: margin-top 160ms;
```

A bit shorter, default easing and no delay. That way the animation maybe still feels part of the fullscreen animation and doesn't call for too much attention.